WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1155,10 +1151,13 @@ - (UIBarButtonItem*)cancelChevronButton
         return _cancelChevronButton;
     }
     
-    WPButtonForNavigationBar* cancelButton = [self buttonForBarWithImageNamed:@"icon-posts-editor-chevron"
-                                                                        frame:NavigationBarButtonRect
-                                                                       target:self
-                                                                     selector:@selector(cancelEditing)];
+    UIImage *image = [UIImage imageNamed:@"icon-posts-editor-chevron"];
```

Comment:
"icon-posts-editor-chevron" could be replaced by a chevron-left gridicon if we can resize it to be bigger.  The size is 20x30 px so I'm not sure that's feasible with the chevron-left gridicon as it is.